Makefile

```diff
@@ -492,7 +492,6 @@ $(soTargetPatterns6): cargo-init make-always
 	  -u builder \
 	  -w /src/rust/actyx \
 	  -e HOME=/home/builder \
-	  -e ANDROID6=yes \
```

Comment:
the whole soTargetPatterns6 needs to go, removing .so files from the APKs as a result (making them smaller)

Reply:
Handled in https://github.com/Actyx/Actyx/pull/538/commits/897edc2fdbc87e894ccff6132486ca19c7b9ea4d